src/clj/game/core/flags.clj

```diff
@@ -253,11 +253,13 @@
   (card-is? card :type type))
 
 (defn has-subtype?
-  "Checks if the specified subtype is present in the card."
+  "Checks if the specified subtype is present in the card, ignoring case."
   [card subtype]
-  (or (has? card :subtype subtype)
-      (when-let [persistent-subs (-> card :persistent :subtype)]
-        (includes? persistent-subs subtype))))
+  (letfn [(contains-sub? [card]
+            (when-let [subs (:subtype card)]
+              (includes? (lower-case subs) (lower-case subtype))))]
```

Comment:
Let us pray that this is the intended meaning of subtypes.

Reply:
Considering that's how icebreakers interact with ice (all subtypes are lower-case on icebreakers and capitalised on ice), I sure hope so 😛